transit/rest_api/urls.py

```diff
@@ -8,17 +8,35 @@
 from rest_framework_simplejwt.views import TokenRefreshView, TokenVerifyView, TokenObtainPairView
 
 
+from transit.rest_api.permissions import IsUserManager
+
+
 class UserSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = User
+        fields = ['url', 'username', 'email', 'is_staff', 'password']
+        extra_kwargs = {'password': {'write_only': True}}
+
+    def create(self, validated_data):
+        user = User(**validated_data)
+        user.set_password(validated_data['password'])
+        user.save()
+        return user
+
+
+class UserUpdateSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
We have to move both user serializers and userviewset to separate .py file or subpackage, those shouldn't be stored in  urls.py

Reply:
ok, I will move. I also thought that this was temporary solutions (what you did previously)